public/data/index.php

```diff
@@ -0,0 +1,8 @@
+<h2>Data</h2>
+<ul>
+<?php
+foreach (glob("*.php") as $filename) {
+    echo "<li><a href='data/{$filename}'>{$filename}</a></li>";
+}
+?>
+</ul>
```

Comment:
do we really want to expose these files to the user? Since those are classes, user would access those directly, only through the API/VIEW files that include them. 
I will show u tomorrow how to protect those so users cant accidentally browse into them